### PR TITLE
Features

### PR DESCRIPTION
**Pull request by wuyepabdul**
In this project, the following were implemented:

- Created a new Rails app called 'hello-rails-react'.
- Initialized project with Git.
- Set up the project with webpacker the react-rails gems as described in the previous lesson's tutorial.
- Made sure that the project has a Postgres database set up.
- Created a table for storing your messages and created 5 different greetings.
- Created an API endpoint that selects a random greeting from my table 
- Created a static view that is the root of my app.
- Created the App component with react-router and rendered it in my static view.
- Created the Greeting component that will display a greeting. Set it up as a route in my App component.
- Created a store, action, and a reducer that will connect to my API endpoint to get the random greeting.
- Displayed the random greeting in my Greeting component.

### CODE REVIEW

File: app/javascript/components/App.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+const store = configureStore();
+
+class App extends React.Component {
+  render() {
+    return (
+      <Provider store={store}>
+        <BrowserRouter>
+          <Routes>
+            <Route path="/" element={<Greeting />} />
+            {/* <Route path="/greeting" element={<Greeting greeting={"greeting Friend "} />} /> */}
```
Comment (rukundoeric) [inline app/javascript/components/App.js]: - [x] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes.

File: app/javascript/configureStore.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  );
+  return store;
+}
+
+// const configureStore = createStore(
+//   reducers,
+//   composeWithDevTools(applyMiddleware(...middleWares)),
+// );
+
+// export default configureStore;
```
Comment (rukundoeric) [inline app/javascript/configureStore.js]: - [x] <b>REQUIRED:</b>The same thing goes here, to make sure that your codebase is always clean, remember to remove commented-codes.

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @wuyepabdul, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 

## Required Changes ♻️
  - [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
  - [x] Check the comments under the review.

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by rukundoeric — APPROVED** [review]
Hi @wuyepabdul,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by wuyepabdul** [comment]
Thanks for the review @rukundoeric

**Pull request merged**